honeybot/plugins/dictionary.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+[joke.py]
+Dictionary Plugin
+
+[Author]
+Nishant, JPMorgan Chase & Co.
+
+[About]
+sends the meaning of the word
+
+[Commands]
+>>> .dictionary <<word>>
+returns a random joke
```

Comment:
a random word?

Reply:
word should be provided , not any random word